README.md

```diff
@@ -24,6 +24,8 @@
     </a>
 	<img alt="Style: Prettier 🧹" src="https://img.shields.io/badge/style-prettier_🧹-21bb42.svg" />
     <img alt="TypeScript: Strict 💪" src="https://img.shields.io/badge/typescript-strict_💪-21bb42.svg" />
+    <img alt="npm package version" src="https://img.shields.io/npm/v/create-typescript-app?color=21bb42" />
+  
```

Comment:
```suggestion
```

Small thing, no need for a newline here I think.

Reply:
Fixed